### PR TITLE
Entry level production config docs

### PR DESCRIPTION
**Pull request by dnil**
Update docs for new production level users.

**How to test**:
1. apply suggestions from docs? Or just read and think?

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,9 +113,9 @@ This is an example of the config file:
 # list of email addresses to send errors to in production
 ADMINS = ['paul.anderson@magnolia.com']
 
-MONGO_HOST = 'localhost'
+MONGO_HOST = 'localhost'		
 MONGO_PORT = 27017
-MONGO_DBNAME = 'scoutTest'
+MONGO_DBNAME = 'scout'
```
Comment (northwestwitch) [inline README.md:118]: 👍

File: docs/admin-guide/server.md [hunk 2]
```diff
@@ -91,4 +106,15 @@ gunicorn \
...
 
+If you are running a larger environment, where this is one component, we encourage a reverse proxy configuration where Scout is served by gunicorn, and reverse proxied by NGIX. Then NGIX will handle the secure communication, and the start config reduces to something like
+```bash
+SCOUT_CONFIG="/full/path/to/scout_flask_conf.py" \
+gunicorn \
+    --workers=4 \
+    --bind="HOST:PORT" \
+    --access-logfile=-
+    --error-logfile=-
+    wsgi_gunicorn:app
```
Comment (northwestwitch) [inline docs/admin-guide/server.md:117]:
```suggestion
    scout.server.auto:app
```

**Review by northwestwitch — APPROVED** [review]
👍 

I have just a suggestion for renaming the wsgi_gunicorn thing to the file that is actually in scout.

**Comment by dnil** [comment]
You need to remind me to update CHANGELOG! 😆

File: docs/admin-guide/server.md (hunk 2 above)
Comment (dnil) [reply docs/admin-guide/server.md:117]: Hm, but we do recast it as wsgi in the app (you know, the proxy fix stuff)? We can test: my guess is both will actually work.

Comment (northwestwitch) [reply docs/admin-guide/server.md:117]: Yes but the question from the user was about this, and the fact that the file was not found..

**Comment by northwestwitch** [comment]
> You need to remind me to update CHANGELOG!

I was testing you, and you passed! 🥇

**Comment by northwestwitch** [comment]
> I was testing you, and you passed! 🥇

No, seriously, I forgot

File: docs/admin-guide/server.md (hunk 2 above)
Comment (dnil) [reply docs/admin-guide/server.md]: Right you are, ofcourse! We should also remove the corresponding from the servers repo..

**Pull request merged**